### ISSUE TITLE
Filter Menu does not stay open

### ISSUE THREAD

**Issue by TonyBarganski** [sodenn/2do-txt#316]
This is a feature request.

On the desktop version the bubble filter screen 'covers-up' the 'task list'. 
I presume this is a hang-up from the mobile phone version where screen space is limited?

#### Desired Outcome
I should be able to see my tasks at the same time, 'while' I have my filter menu open and am making changes to filters. 
I should see the results of my filter changes in real-time on the right pane.

In other words, if I open the Filter Menu, it does not cover over the Task List but appear to the left of the Task List.
